Tests/KituraTests/TestCodableRouter.swift

```diff
@@ -39,7 +39,8 @@ class TestCodableRouter: KituraTest {
             ("testRouteParameters", testRouteParameters),
             ("testCodableRoutesWithBodyParsingFail", testCodableRoutesWithBodyParsingFail),
             ("testCodableGetQueryParameters", testCodableGetQueryParameters),
```

Comment:
Please rename this test to "testCodableGetSingleQueryParameters", then follow it with your new "testCodableGetArrayQueryParameters" test, then finish the array with the existing "testCodableDeleteQueryParameters" test.

That way the test naming is consistent with the non-queryparams cases.